pootle/templates/staticpages/_body.html

```diff
@@ -9,7 +9,14 @@
   {% endblocktrans %}
   </div>
   {% endif %}
-  <h1 class="title">{{ page.title }}</h1>
+  <h1 class="title">
+    {{ page.title }}
+    {% if user.is_superuser %}
+    <span class="edit-link">
```

Comment:
Tested this on my computer, but the edit link is not displayed with smaller font. I refreshed several times and dropped cache. Perhaps it is only me, so I would like someone else to test and confirm this is displayed as expected.

I also considered this to be not in the header line since some headers can be long, but I don't think it is a major issue. In other words, this is something I can live with.